buildpacks/nodejs-engine/src/main.rs

```diff
@@ -85,30 +88,32 @@ impl Buildpack for NodeJsEngineBuildpack {
             Requirement::parse(LTS_VERSION).expect("The default Node.js version should be valid")
         };
 
-        let target_release =
-            inv.resolve(&version_range)
-                .ok_or(NodeJsEngineBuildpackError::UnknownVersionError(
-                    version_range.to_string(),
-                ))?;
+        let target_artifact = match (consts::OS.parse::<Os>(), consts::ARCH.parse::<Arch>()) {
```

Comment:
Are `consts::OS` and `consts::ARCH` equivalent to the `context.target.os` and `context.target.arch` provided by [libcnb.rs](https://github.com/heroku/libcnb.rs/blob/main/libcnb/src/target.rs)?

Reply:
No, the `Target` fields set by libcnb.rs maps to the environment variables provided by the lifecycle (and in turn, read from the run image): https://buildpacks.io/docs/for-buildpack-authors/concepts/targets/

The values will often be equivalent though (but may differ in some scenarios, e.g. when the build and run images have different architectures), but that's as far as I'm aware that won't happen (at least for buildpacks that require OS/arch specific binaries).